electron/main.js

```diff
@@ -168,6 +169,20 @@ function showMainWindow() {
     main.setBounds(init.restore('bounds', main.getBounds()));
   }
 
+  main.webContents.session.setCertificateVerifyProc((hostname = '', certificate = {}, cb) => {
+    const {data: cert = ''} = certificate;
+
+    if (certutils.hostnameShouldBePinned(hostname) === true) {
+      if (certutils.verifyPinning(hostname, cert) === true) {
```

Comment:
we are in control of these functions.. so it's either `true` or `false` so no need to `=== true`